services/catalog/src/simcore_service_catalog/services/director.py

```diff
@@ -18,7 +18,7 @@ def setup_director(app: FastAPI) -> None:
     # init client-api
     logger.debug("Setup director at %s...", settings.base_url)
     app.state.director_api = DirectorApi(
-        base_url=settings.base_url, vtag=app.state.settings.director.vtag
+        base_url=settings.base_url, vtag=app.state.settings.director.vtag, app=app
```

Comment:
minor: hmm why you pass the app? you have the settings just above. if you pass the full app, then you could also change the full signature.
1st and 2nd arguments are already giving sub parts of the settings

Reply:
You are correct. Got confused by something similar, but different, elsewhere.